spec/reek/spec/should_reek_of_spec.rb

```diff
@@ -54,23 +54,6 @@
     end
   end
 
-  context 'checking code in a Dir' do
-    before :each do
-      @clean_dir = Dir['spec/samples/three_clean_files/*.rb']
-      @smelly_dir = Dir['spec/samples/two_smelly_files/*.rb']
-      @matcher = Reek::Spec::ShouldReekOf.new(:UncommunicativeVariableName,
-                                              name: '@s')
-    end
-
-    it 'matches a smelly String' do
-      expect(@matcher.matches?(@smelly_dir)).to be_truthy
-    end
-
-    it 'doesnt match a fragrant String' do
-      expect(@matcher.matches?(@clean_dir)).to be_falsey
-    end
-  end
```

Comment:
Because we're not allowing passing a dir to `Examiner.new` anymore, the reek_of matchers also don't support passing a directory anymore.